.travis.yml

```diff
@@ -82,13 +102,22 @@ matrix:
       - JOB_TAG=_NUMPY_DEV
       - NOSE_ARGS="not slow and not network and not disabled"
       - PANDAS_TESTING_MODE="deprecate"
+      addons:
+        apt:
+          packages:
```

Comment:
I fixed this, it was caused by not indenting the new section correcting in the travis file in the allow failures section.